### PR TITLE
DEV: update code owners for the docs

### PR DESCRIPTION
**Pull request by samir-nasibli**
## Description

update code owners for the docs
---

PR should start as a draft, then move to ready for review state after CI is passed and all applicable checkboxes are closed.
This approach ensures that reviewers don't spend extra time asking for regular requirements.

You can remove a checkbox as not applicable only if it doesn't relate to this PR in any way.
For example, PR with docs update doesn't require checkboxes for performance while PR with any change in actual code should have checkboxes and justify how this code change is expected to affect performance (or justification should be self-evident).

Checklist to comply with **before moving PR from draft**:

**PR completeness and readability**

- [x] I have reviewed my changes thoroughly before submitting this pull request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes or created a separate PR with update and provided its number in the description, if necessary.
- [x] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_.
- [x] I have added a respective label(s) to PR if I have a permission for that.
- [x] I have resolved any merge conflicts that might occur with the base branch.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,9 +4,9 @@
 .circleci/                    @napetrov @Alexsandruss @homksei @ahuber21 @ethanglaser
 
 # Docs
-*.md                          @maria-Petrova @napetrov @emmwalsh @Alexsandruss @samir-nasibli @icfaust
-doc/                          @maria-Petrova @napetrov @emmwalsh @Alexsandruss @samir-nasibli @icfaust
-requirements-doc.txt          @emmwalsh @Alexsandruss
+*.md                          @maria-Petrova @napetrov @emmwalsh @Alexsandruss @samir-nasibli @icfaust @david-cortes-intel
```
Comment (samir-nasibli) [inline .github/CODEOWNERS:7]: It seems we need to remove emmwalsh

Comment (ethanglaser) [reply .github/CODEOWNERS:7]: Yes, she should be removed

**Review by icfaust — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -4,9 +4,9 @@
 .circleci/                    @napetrov @Alexsandruss @homksei @ahuber21 @ethanglaser
 
 # Docs
-*.md                          @maria-Petrova @napetrov @emmwalsh @Alexsandruss @samir-nasibli @icfaust
-doc/                          @maria-Petrova @napetrov @emmwalsh @Alexsandruss @samir-nasibli @icfaust
-requirements-doc.txt          @emmwalsh @Alexsandruss
+*.md                          @maria-Petrova @napetrov @Alexsandruss @samir-nasibli @icfaust @david-cortes-intel
+doc/                          @maria-Petrova @napetrov @Alexsandruss @samir-nasibli @icfaust @david-cortes-intel
+requirements-doc.txt          @Alexsandruss @samir-nasibli @icfaust @david-cortes-intel
```
Comment (ethanglaser) [inline .github/CODEOWNERS:9]: Minor comment but Samir and Alexander would already be covering requirements-doc.txt from line 20

Comment (samir-nasibli) [reply .github/CODEOWNERS:9]: The last matching pattern takes the most precedence. From my understanding, when someone opens a pull request that only modifies `requirements-doc.txt` file, only the owners from line 9 and not the global owner(s) for the `requirements*` from L20 will be requested for a review.

**Pull request merged**